tools/libSwiftSyntaxParser/libSwiftSyntaxParser.cpp

```diff
@@ -37,9 +37,26 @@ typedef swiftparse_trivia_piece_t CTriviaPiece;
 typedef swiftparse_syntax_kind_t CSyntaxKind;
 
 namespace {
+
+unsigned getByteOffset(SourceLoc Loc, SourceManager &SM, unsigned BufferID) {
```

Comment:
Nitpick but I much prefer that `static` functions have `static` in front of them. Otherwise it is not obvious to me that the functions are static unless I check that they are contained in an anonymous namespace, which can be far apart from the function declaration. Same for the function below.